app/assets/stylesheets/itemcard.scss

```diff
@@ -1,15 +1,19 @@
 .itemcard{
-    border: none;
+    padding: 0;
+}
+
+.itemcard .card-body{
+    padding: 0;
 }
 
 .itemcard .card-body img{
     border-radius: 5px;
     object-fit: cover;
-    width: 300px;
     height: 300px;
```

Comment:
I personally don't know how we could make this more responsive (make the images scale in both directions simultaneously)

Reply:
What behavior do you want to achieve? :)

Reply:
I want to achieve the following things:
- The image should be square (currently it's *almost* square as the card is a little bit wider than 300px)
- The image should be cropped (implemented by the `object-fit` property)
- The image should fill the whole width of the card (implemented by removing the `width` property and some styling in other selectors)
- **I don't want to hard-code a specific height (so that a card resizing (by the users' device or by us) doesn't make the whole UI ugly, but bootstrap does not seem to resize the cards)**

The current implementation might be good enough, especially as bootstrap seems to keep the card size constant. Your thoughts?

Reply:
A quick research got me to this:

https://stackoverflow.com/questions/23400232/force-bootstrap-responsive-image-to-be-square

If you want to use plain css, use `object-fit` and `width: 100%` and `height: 100%` for the image. Because you have to give the itemcard a size, e.g. 30rem or 300px, you have to use hard-code width/height or [`.container`](https://getbootstrap.com/docs/5.0/layout/containers/).